keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/ClusterSpec.kt

```diff
@@ -117,16 +119,18 @@ data class ClusterSpec(
   private val _defaults: ServerGroupSpec,
   @JsonInclude(NON_EMPTY)
   val overrides: Map<String, ServerGroupSpec> = emptyMap(),
-  @JsonIgnore
-  override val deliveryArtifact: DeliveryArtifact? = null,
-  @JsonIgnore
+  @JsonProperty(access = Access.WRITE_ONLY)
+  override val artifactName: String? = null,
+  @JsonProperty(access = Access.WRITE_ONLY)
+  override val artifactType: ArtifactType? = ArtifactType.deb,
```

Comment:
Artifact type is now assumed here, since this spec can only really be associated with that type.

Reply:
is there anything that validates that?

Reply:
Not sure what you mean. It's hard-coded here, what validation would we have elsewhere?

Reply:
you said "this spec can only really be associated with that type" - I guess i was wondering what a failure looks like. But, thinking about it now, we just wouldn't be able to find the right package when baking or image when searching.  so the failure would be more of a generic "this didn't work i can't find anything"